cordova-js-src/splashscreen.js

```diff
@@ -203,10 +212,30 @@ function enableUserInteraction() {
     document.body.style['-ms-content-zooming'] = origZooming;
 }
 
+// Make title bg color match splashscreen bg color
+function colorizeTitleBar() {
+    var appView = Windows.UI.ViewManagement.ApplicationView.getForCurrentView();
+    if (appView.titleBar) {
+        titleInitialBgColor = appView.titleBar.backgroundColor;
+
+        appView.titleBar.backgroundColor = titleBgColor;
+        appView.titleBar.buttonBackgroundColor = titleBgColor;
+    }
+}
+
+// Revert title bg color
+function revertTitleBarColor() {
+    var appView = Windows.UI.ViewManagement.ApplicationView.getForCurrentView();
+    if (appView.titleBar) {
```

Comment:
The change only affects desktop projects and this check for `appView.titleBar` is for Windows 8.1 OS only where titlebar isn't supported (store apps are shown without chrome).